src/pudl_archiver/metadata/sources.py

```diff
@@ -44,4 +145,121 @@
         "license_pudl": LICENSES["cc-by-4.0"],
         "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
     },
+    "usgsuswtdb": {
+        "title": "USGS USWTDB - U.S. Wind Turbine Database",
+        "path": "https://energy.usgs.gov/uswtdb/data/",
```

Comment:
Use the same top-level URL pattern here as is used for the PV dataset below (i.e. no `data/` at the end).

```suggestion
        "path": "https://energy.usgs.gov/uswtdb/",
```